### PR TITLE
OP-15: wildcard cert configuration

### PR DESCRIPTION
**Pull request by jkapponms**

### CODE REVIEW

File: charts/opennms-operator/values.yaml [hunk 1]
```diff
@@ -12,4 +12,18 @@ operator:
...
 customErrors:
   image: quay.io/kubernetes-ingress-controller/custom-error-pages-amd64:0.4
-  debug: true
\ No newline at end of file
+  debug: true
+ssl:
+  enabled: true
+  issuer:
+    issuerServer: https://acme-v02.api.letsencrypt.org/directory
+    issuerUserEmail: email@address.com
```
Comment (bborovan) [inline charts/opennms-operator/values.yaml:20]: Is this valid for a user email?

**Review by bborovan — COMMENTED** [review]
See my comment regarding the issuerUserEmail... not sure if that is valid?

**Comment by jbeuree** [comment]
Will this work to request a wildcard cert and share it amongst resources, or will it still use the cert replication? I think when this was originally attempted, it ended up requesting the same wildcard cert many times.

File: charts/opennms-operator/values.yaml (hunk 1 above)
Comment (jkapponms) [reply charts/opennms-operator/values.yaml:20]: No it isn't. All of these values are just placeholders to be filled in by whomever uses the operator's helm charts.

**Comment by jkapponms** [comment]
@jbeuree 
Cert replication, one cert will be replicated into each instance.

**Review by bborovan — APPROVED** [review]

**Comment by jaymberry** [comment]
The info in charts/opennms-operator/values.yaml  is off, will this be overwrittern?

**Comment by jkapponms** [comment]
> The info in charts/opennms-operator/values.yaml is off, will this be overwrittern?

Yes, it's meant to be overwritten by whomever uses it.

File: charts/opennms-operator/values.yaml [hunk 2]
```diff
@@ -12,4 +12,18 @@ operator:
...
+  cert:
+    hostname: "*.wildcard.domain.com"
+  solver:
+    # Configure according to: https://cert-manager.io/docs/configuration/acme/dns01/
```
Comment (jaymberry) [inline charts/opennms-operator/values.yaml:24]: Does the info here get overwritten?

**Review by jaymberry — APPROVED** [review]

**Review by jaymberry — APPROVED** [review]
looks good

**Review by jbeuree — APPROVED** [review]

**Pull request merged**